pkg/scd/store/cockroach/store.go

```diff
@@ -15,81 +17,61 @@ var (
 	DefaultClock = clockwork.NewRealClock()
 )
 
-// Store is an implementation of scd.Store using
+// repo is an implementation of repos.Repo using
 // a CockroachDB transaction.
-type Store struct {
-	tx     *sql.Tx
-	logger *zap.Logger
-	clock  clockwork.Clock
-}
-
-// Transaction is an implementation of scd.Transaction using
-// a CockroachDB transaction.
-type Transaction struct {
-	tx     *sql.Tx
+type repo struct {
+	q      dsssql.Queryable
 	logger *zap.Logger
 	clock  clockwork.Clock
 }
 
-// Implement store.Transaction interface
-func (t *Transaction) Store() (scdstore.Store, error) {
-	return &Store{
-		tx:     t.tx,
-		logger: t.logger,
-		clock:  t.clock,
-	}, nil
-}
-
-// Implement store.Transaction interface
-func (t *Transaction) Commit() error {
-	return t.tx.Commit()
-}
-
-// Implement store.Transaction interface
-func (t *Transaction) Rollback() error {
-	return t.tx.Rollback()
-}
-
-// Transactor is an implementation of scd.Transactor using
+// Store is an implementation of an scd.Store using
 // a CockroachDB database.
-type Transactor struct {
+type Store struct {
 	db     *cockroach.DB
 	logger *zap.Logger
 	clock  clockwork.Clock
 }
 
-// NewTransactor returns a Transactor instance connected to a cockroach instance via db.
-func NewTransactor(db *cockroach.DB, logger *zap.Logger) *Transactor {
-	return &Transactor{
+// NewStore returns a Store instance connected to a cockroach instance via db.
+func NewStore(db *cockroach.DB, logger *zap.Logger) *Store {
+	return &Store{
 		db:     db,
 		logger: logger,
 		clock:  DefaultClock,
 	}
 }
 
-// Implement store.Transactor interface
-func (t *Transactor) Transact() (scdstore.Transaction, error) {
-	tx, err := t.db.Begin()
-	if err != nil {
-		return nil, err
-	}
-	return &Transaction{
-		tx:     tx,
-		logger: t.logger,
-		clock:  t.clock,
+// Interact returns a new repo instance.
```

Comment:
nit: I think it would be best to comment interface implementations by referring to the interface rather than describing what the function does to prevent comment inconsistency and to make it clear that the function is implementing an interface.  So, here: "Interact implements store.Interactor interface."